### DISCUSSION TITLE
About reports

### DISCUSSION THREAD

**Discussion by williamhrs** [parcelvoy/platform#478] [Q&A]
Are there plans to implement better reports on the software? We just deployed an instance of it, the software looks very promising, and reports is something that we use a LOT

**Comment by pushchris** [comment]
@williamhrs are there any reports you are specifically looking for? We currently don't have any plans for adding in additional reporting. In general we've found most companies these days already have some unified event reporting software they are using (Mixpanel, Amplitude, Posthog, etc) and would rather metrics live there so they can use their other event data when generating reports.

**Comment by williamhrs** [comment]
We're trying to migrate from ActiveCampaign, and for reports, I'm  talking about e-mail reports, with ability to filter for dates, automations, blasts. 

We'll probably try to figure something out on our end, but I thought I'd ask here! :D

**Comment by pushchris** [comment]
By reports it sounds like what you actually mean is something exportable to like a CSV? Do you have any examples of reports you've found useful in the past? What sort of data are you trying to get at?

**Comment by williamhrs** [comment]
![image](https://github.com/user-attachments/assets/5ac323ae-0715-410f-8372-d77b2f299fb0)
for example:

https://help.activecampaign.com/hc/en-us/categories/360000293350-Reports

**Comment by leobarcellos** [comment]
This PR #523 doesn't necessarily solves this discussion, but it's helpful.

I think that a "Reports" view with more detailed info is still needed

**Comment by williamhrs** [comment]
As for reports, it's really important for us, especially for "trigger" campaigns to monitor date periods, let's say for example I have a welcome email.

I have an overwall open rate of 50% on it. 

But After let's say a week ago, it started not working anymore. Since we have the full length of data, the overall open rate will take quite a while to go down, since we're seeing an average of the full time period. 

(Sorry if Sometimes I'm not clear enough, since english is not my main language, sometimes it can be hard to explain technical stuff lol)

**Comment by williamhrs** [comment]
@pushchris comming back to this topic

![image](https://github.com/user-attachments/assets/f8b61dd8-8f23-47a0-abed-1615732e7b2b)


This trigger email for example, if I can't filter the metrics to a certain period I can't know when the e-mail started to perform poorly.

I can only see data for the whole period, there's no way to filter it for something like "last 5 days" to see how it's performing. 

when we have a high volume email, if it started performing bad, it can take a while for us to figure it out, since we can only see metrics for the whole period.

**Comment by williamhrs** [comment]
Just to illustrate a little more ->

I created a looker report using data direct from the database -> I'm not much of a developer these days so I don't really touch the code. 

![image](https://github.com/user-attachments/assets/638717a9-6057-4b4f-ab48-bc7d6aef7eda)
